common/djangoapps/xblock_django/user_service.py

```diff
@@ -58,6 +61,10 @@ def __init__(self, django_user, **kwargs):
         self._anonymous_user_id = kwargs.get('anonymous_user_id', None)
         self._deprecated_anonymous_user_id = kwargs.get('deprecated_anonymous_user_id', None)
         self._request_country_code = kwargs.get('request_country_code', None)
+        self._user_has_manage_content = kwargs.get('user_has_manaege_content', False)
```

Comment:
typing error in 'user_has_manage_content'
```suggestion
        self._user_has_manage_content = kwargs.get('user_has_manage_content', False)
```